pyproject.toml

```
@@ -67,7 +67,7 @@ testing = [
     "pytest-cov",
     "pytest-timeout",
     "flexmock",
-    "deepdiff",
+    "deepdiff == 7.0.1",
```

This would break Packit RPMs.

```suggestion
    "deepdiff <= 7.0.1",
```